### PR TITLE
Add and elaborate on comments in config.cpp

### PR DESCRIPTION
**Pull request by yummydirtx**
There were areas that weren't documented fully or at all so I touched them up.

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/5473)

### CODE REVIEW

**Comment by wwylele** [comment]
https://www.reddit.com/r/ProgrammerHumor/comments/8w54mx/code_comments_be_like/

File: src/citra/config.cpp [hunk 1]
```diff
@@ -71,7 +72,7 @@ static const std::array<std::array<int, 5>, Settings::NativeAnalog::NumAnalogs>
 }};
 
 void Config::ReadValues() {
-    // Controls
+    // These are the controls
```
Comment (wwylele) [inline src/citra/config.cpp:75]: These three comment in `ReadValues`  actually share the same hidden verb: "Load the XXX-related configuration values from INI to Settings, so 
 - they should be the same when you append them
 - "Define" is not an accurate term,

**Comment by BreadFish64** [comment]
Hi, it's great that you're trying to help, but this in particular doesn't do much. The places where you added comment are self-documenting and using complete sentences is not necessary. It's better to make code obvious by naming things well, and only use comments to describe complex behavior and interfaces. In my opinion these places are already understandable the way they are.

**Comment by yummydirtx** [comment]
> Hi, it's great that you're trying to help, but this in particular doesn't do much. The places where you added comment are self-documenting and using complete sentences is not necessary. It's better to make code obvious by naming things well, and only use comments to describe complex behavior and interfaces. In my opinion these places are already understandable the way they are.
> For example, the parsing error comment is redundant since it's followed by a warning log which explains the error.

I do agree with you on some points, but I do believe it is helpful to put comments, even if they seem obvious. Someone reading through source code is looking out for comments to explain certain blocks of code. This would help them to know where to look as compared to mentally skipping a section.

File: src/citra/config.cpp (hunk 1 above)
Comment (yummydirtx) [reply src/citra/config.cpp:75]: > These three comment in `ReadValues`  actually share the same hidden verb: "Load the XXX-related configuration values from INI to Settings" where XXX is the existing comment, so 
> 
>  - they should be the same when you append them
> 
>  - "Define" is not an accurate term,

It certainly could be accurate to say that they are taking already existing values to define those things

**Pull request closed without merging**